### ISSUE TITLE
RPC API for IOTA Wallet

### ISSUE THREAD

**Issue by i3bitcoin** [iotaledger/wallet#1190]
Is there any RPC Api available for IOTA Wallet?

**Comment by rajivshah3** [comment]
What do you mean by RPC?

**Comment by i3bitcoin** [comment]
I mean Json-RPC or Rest API

**Comment by rotilho** [comment]
You can connect to the IRI. Find more details: https://iota.readme.io/reference

**Comment by i3bitcoin** [comment]
I'm searching the wallet daemon with it's own RPC Api to interact with IRI, that can generate addresses, find transactions and send funds. There is a cli-app (https://github.com/iotaledger/cli-app) that has all functions I need but it doesn't have rpc api.

**Comment by i3bitcoin** [comment]
@skinderis nope, at least I couldn't find any.... it seems that nobody cares about merchants. I have started developing rpc over this cli https://github.com/iotaledger/cli-app by myself but it's still under development. At least thanks to @mark-bradshaw for coding it.

**Comment by cdlgz** [comment]
> @skinderis nope, at least I couldn't find any.... it seems that nobody cares about merchants. I have started developing rpc over this cli https://github.com/iotaledger/cli-app by myself but it's still under development. At least thanks to @mark-bradshaw for developing it.

@i3bitcoin Is your development done? Could you open source it on github?

**Comment by fijter** [comment]
Have you seen https://github.com/iotaledger/rpchub? I think it does everything you want and more.
